docs/sql/statements/create-foreign-table.rst

```diff
@@ -0,0 +1,65 @@
+.. highlight:: psql
+.. _ref-create-foreign-table:
+
+========================
+``CREATE FOREIGN TABLE``
+========================
+
+Create a foreign table
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+Synopsis
+========
+
+.. code-block:: psql
+
+  CREATE FOREIGN TABLE [ IF NOT EXISTS ] table_name ([
```

Comment:
Mostly because that's what was used in https://www.postgresql.org/docs/current/sql-createforeigntable.html

Can change it if it's important.

Reply:
I think, `table_ident` reflects more that it can include a `schema_name` as well. Otherwise I do not really care, beside that I think it's good to be consistent. I think, PG always uses `table_name` while we use `table_ident`.

Reply:
Changed it